components/TARGET_PSA/TARGET_TFM/COMPONENT_SPE/secure_fw/spm/spm_db_setup.h

```diff
@@ -38,7 +38,7 @@ struct spm_partition_db_t {
         data.partition_priority = TFM_PRIORITY(priority);                     \
     } while (0)
 
-#if (TFM_LVL == 1) && !defined(TFM_PSA_API)
+#if (TFM_LVL == 1)
```

Comment:
Please comment in our patch why this change is needed.

Reply:
done